src/js/samples/querys.ts

```diff
@@ -0,0 +1,80 @@
+import { forEach, map, reject, union } from "lodash-es/lodash";
+import { useMutation, useQuery, useQueryClient } from "react-query";
+import { listSamples, update } from "./api";
+import { SampleMinimal } from "./types";
+
+type Labels = {
+    /** The hex encoded color */
+    color: string;
+    /** The detailed description */
+    description: string;
+    /** The unique identifier */
+    id: number;
+    /** The display name */
+    name: string;
+    /** Whether all selected samples contain the label */
+    allLabeled: boolean;
+};
```

Comment:
```suggestion
type SampleLabel =Label & {
    allLabeled: boolean;
};
```
My suggested name isn't amazing, but makes a bit more clear it is not a standard label.